network/ssh.go

```diff
@@ -128,8 +128,9 @@ func ensurePortSuffix(host string, port int) string {
 
 func (a *SSHAgent) newClient(host string, user string, auth []ssh.AuthMethod) (*ssh.Client, error) {
 	sshcfg := ssh.ClientConfig{
-		User: user,
-		Auth: auth,
+		User:            user,
+		Auth:            auth,
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
```

Comment:
Are we sure this is what we want here in all cases? https://godoc.org/golang.org/x/crypto/ssh#InsecureIgnoreHostKey